catalog.html

```diff
@@ -0,0 +1,263 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Техномарт-каталог</title>
+  </head>
+  <body>
+    <header class="main-header">
+      <nav class="site-navigation">
+        <p class="main-logo">Техномарт</p>
+          <ul>
+            <li><input type="search">Поиск</li>
+            <li><button>Закладки</button></li>
```

Comment:
```suggestion
            <li><button type="button">Закладки</button></li>
```
Вот спека: https://www.w3.org/TR/html/sec-forms.html#the-button-element
А вот хорошее объяснение из подготовительного задания: https://up.htmlacademy.ru/htmlcss/25/module/1/prepare/4#description